package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "lugdbench-api",
+  "version": "1.0.0",
+  "description": "",
+  "engines": {
+    "node": ">=6.10x"
+  },
+  "main": "server.js",
+  "scripts": {
+    "start": "node server.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Lugdunum3D",
+  "license": "ISC",
```

Comment:
+ add https://github.com/Lugdunum3D/glTF2-loader/blob/master/LICENSE at the root, replacing glTF2-loader with LugBench-API